data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml

```diff
@@ -5,9 +5,9 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-
     <fileDesc>
       <titleStmt>
         <!--This is the overall file title. It's not really used.-->
-        <title>Carmina</title>
+        <title xml:lang="lat">Carmina</title>
         <title type="sub">Machine readable text</title>
-        <author>C. Valerius Catullus</author>
+        <author xml:lang="lat">C. Valerius Catullus</author>
         <editor role="editor" n="Burton">Sir Richard Francis Burton</editor>
```

Comment:
This is wrong. Generally translations need a translator, not an editor. So it is `editor role="translator`" and the n attribute is not needed. This is one of those things that jump out at you when work on lots of headers as the old files are completely inconsistent on this.